docs/Makefile

```diff
@@ -20,7 +20,7 @@ I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) source
 .PHONY: help clean html dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck doctest gettext all default
 
 default:
-	$(SPHINXBUILD) -b html -nW $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	$(SPHINXBUILD) -b html -n $(ALLSPHINXOPTS) $(BUILDDIR)/html
```

Comment:
I don't think we should change the Makefile as part of this PR, should we?

Reply:
Ok, so we will then just ignore the failing docs check, correct?

Reply:
Yes.

Reply:
This is already done in `docs-revamp` you should just rebase this branch on the latest to make sure it is up to date. All builds have passed so far

Reply:
> All builds have passed so far

How do you mean? When I enable `W` on my branch, the docs build shows lots of warnings that are completely unrelated to the changes I'm making here (e.g. `docs/source/concepts/processes.rst: WARNING: document isn't included in any toctree`)

Reply:
What I was referring to that the first thing I did on top of `docs-revamp` is remove the `W` flag as you did. So this change has already been part of `docs-revamp` for some time and there is no need to do it here. Just rebase the branch

Reply:
Done!